iOS/MaterialComponents/source/MaterialComponents/Extensions.cs

```diff
@@ -80,80 +80,27 @@ public static class ShadowElevations {
 		public static readonly nfloat Switch = (nfloat)1.0;
 	}
 
-	partial class ShapedView {
-		public ShapedView (CGRect frame) : this (frame, null) {
-		}
-	}
-
-	partial class BottomAppBarColorThemer {
-		// renamed/obsolete members
-		[Obsolete ("Use ApplySurfaceVariant instead.")]
-		public static void ApplySurfaceVariantWithSemanticColorScheme (IColorScheming colorScheme, BottomAppBarView bottomAppBarView) {
-			ApplySurfaceVariant (colorScheme, bottomAppBarView);
-		}
-	}
-
 	partial class TextFieldColorThemer {
-		public static void ApplySemanticColorSchemeToAll (IColorScheming colorScheme, Type textInputControllerType) {
-			ApplySemanticColorSchemeToAll (colorScheme, new Class (textInputControllerType));
-		}
 		public static void ApplySemanticColorSchemeToAll<T> (IColorScheming colorScheme) where T : ITextInputController {
 			ApplySemanticColorSchemeToAll (colorScheme, typeof (T));
 		}
-
-		// renamed/obsolete members
-		[Obsolete ("Use ApplySemanticColorScheme instead.")]
-		public static void ApplySemanticColorSchemeToTextInputController (IColorScheming colorScheme, ITextInputController textInputController) {
-			ApplySemanticColorScheme (colorScheme, textInputController);
-		}
-		[Obsolete ("Use ApplySemanticColorScheme instead.")]
-		public static void ApplySemanticColorSchemeToTextInput (IColorScheming colorScheme, ITextInput textInput) {
-			ApplySemanticColorScheme (colorScheme, textInput);
-		}
 	}
 
 	partial class TextFieldFontThemer {
-		public static void ApplyFontSchemeToAll (IFontScheme fontScheme, Type textInputControllerType) {
-			ApplyFontSchemeToAll (fontScheme, new Class (textInputControllerType));
-		}
 		public static void ApplyFontSchemeToAll<T> (IFontScheme fontScheme) where T : ITextInputController {
 			ApplyFontSchemeToAll (fontScheme, typeof (T));
 		}
-
-		// renamed/obsolete members
-		[Obsolete ("Use ApplyFontScheme instead.")]
-		public static void ApplyFontSchemeToTextInputController (IFontScheme fontScheme, ITextInputController textInputController) {
-			ApplyFontScheme (fontScheme, textInputController);
-		}
-		[Obsolete ("Use ApplyFontScheme instead.")]
-		public static void ApplyFontSchemeToTextField (IFontScheme fontScheme, TextField textField) {
-			ApplyFontScheme (fontScheme, textField);
-		}
 	}
 	
 	partial class TextFieldTypographyThemer {
-		public static void ApplyTypographySchemeToAll (ITypographyScheming typographyScheme, Type textInputControllerType) {
-			ApplyTypographySchemeToAll (typographyScheme, new Class (textInputControllerType));
-		}
 		public static void ApplyTypographySchemeToAll<T> (ITypographyScheming typographyScheme) where T : ITextInputController {
```

Comment:
Can you do …?

```csharp
[Static]
[Wrap ("ApplyTypographySchemeToAll (typographyScheme, typeof (T))")]
void ApplyTypographySchemeToAll<T> (ITypographyScheming typographyScheme) where T : ITextInputController;
```

If doesn't work, everything else, LGTM!

Reply:
@dalexsoto Could this be a bug?

Reply:
Nope, Wrap is not intended to be used that way https://github.com/xamarin/xamarin-macios/issues/5444#issuecomment-456086890